app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="tasks_container">
+    <%= form_with model: @task, class: 'create-task' do |f| %>
+      <div class="form-group w-75 p-3">
+        <%= f.label :name %>
+        <%= f.text_field :name, class: "form-control" %>
+      </div>
+      <div class="form-group w-75 p-3">
+        <%= f.label :description %>
+        <%= f.text_area :description, class: "form-control" %>
+      </div>
+      <div class="form-group w-75 p-3">
+        <%= f.label :completed_at %>
+        <%= f.text_field :completed_at, class: "form-control" %>
```

Comment:
because you give this form field to the user, when someone first creates a task it has a value of `''` rather than `nil`. Therefore, when a user first goes to mark a task as complete, they end up changing the task's `completed_at` value from `''` to `nil`.

Either remove this field so that the only way that `completed_at` changes is thru the `mark_complete` route, or update your view to avoid using `.empty?` to check and instead use `.nil?`